.github/workflows/build-and-test.yml

```diff
@@ -224,7 +224,7 @@ jobs:
 
   run-benchmarks:
     name: Run runtime benchmarks
-    runs-on: [performance]
+    runs-on: [self-hosted, performance]
```

Comment:
runtime benchmarks should be only run on specific hardware configuration matching what parity uses, i thing self-hosted are stronger ones, right ?